packages/experiments/src/components/Button/docs/ButtonDos.md

```diff
@@ -1,11 +1,9 @@
-- Make sure the label conveys a clear purpose of the button to the user.
-- Button labels must describe the action the button will perform and should include a verb. Use concise, specific, self-explanatory labels, usually a single word.
-- Buttons should always include a noun if there is any room for interpretation about what the verb operates on.
+- Use Button labels that have a clear purpose. Labels should describe the action of the button in a single line and include a concise verb. If their is room for interpretation about what the verb operates on, a noun should also be used.
 - Consider the effect localization will have on the button and what will happen to components around it.
-- If the button’s label content is dynamic, consider how the button will resize and what will happen to components around it.
+- Consider how the button will resize if the button’s label content is dynamic.
 - Use only a single line of text in the label of the button.
-- Expose only one or two buttons to the user at a time, for example, "Accept" and "Cancel". If you need to expose more actions to the user, consider using checkboxes or radio buttons from which the user can select actions, with a single command button to trigger those actions.
+- Expose only one or two buttons to the user at a time. If you need to expose more actions to the user, consider using checkboxes or radio buttons.
 - Show only one primary button that inherits theme color at rest state. In the event there are more than two buttons with equal priority, all buttons should have neutral backgrounds.
-- "Submit", "OK", and "Apply" buttons should always be styled as primary buttons. When "Reset" or "Cancel" buttons appear alongside one of the above, they should be styled as secondary buttons.
-- Default buttons should always perform safe operations. For example, a default button should never delete.
-- Use task buttons to cause actions that complete a task or cause a transitional task. Do not use buttons to toggle other UI in the same context. For example, a button may be used to open an interface area but should not be used to open an additional set of components in the same interface.
+- Style "Submit", "OK", and "Apply" buttons as primary buttons. Style "Reset" or "Cancel" buttons that appear alongside one of the above, as secondary buttons.
```

Comment:
Remove this. Adds no valuable information,

Reply:
Not so sure about that...maybe it's not valuable to someone who's been doing this awhile, but it could be important for someone newer or who's not actually a designer. (Though the wording should be updated to say "default buttons" instead of "secondary buttons.")

Reply:
We don't want the ask the user to style anything. That would be the wrong guidance.

We could say: Use PrimaryButtons for the primary action (e.g. the "OK" or "Apply" button) and DefaultButtons for all others.

But it already says something similar in another rule (use one primary button)